lang/cs/Org.Apache.REEF.Examples.HelloREEF/Readme.md

```diff
@@ -6,4 +6,25 @@ This project contains a simple example of a REEF Program. It contains the follow
   * `HelloTask`: This Task prints a greeting to STDOUT of the Evaluator.
 
 ## Running it
-Just run the main class, `HelloREEF`, followed by the runtime you want, e.g. `local`.
\ No newline at end of file
+Just run the main class, `HelloREEF`, followed by the runtime you want, e.g. `local`.
```

Comment:
If Evaluatoer.exe doesn't appear, you can update the doc for adding evaluator.exe only and indicate if it is missing from the binary folder, then add it ...  
Not sure it is because it is exe. I actually deleted my bin folder, deleted package folder and then rebuild. I do see evalautor.exe is populated in the REEF.client bin folder.